planner/core/common_plans.go

```diff
@@ -530,10 +530,15 @@ func (e *Explain) prepareOperatorInfo(p PhysicalPlan, taskType string, indent st
 	row := []string{e.prettyIdentifier(p.ExplainID(), indent, isLastChild), count, taskType, operatorInfo}
 	if e.Analyze {
 		runtimeStatsColl := e.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl
-		if runtimeStatsColl.Exists(p.ExplainID()) {
+		copTaskExecDetail := runtimeStatsColl.CopSummary(p.ExplainID())
+		// There maybe some mock information for cop task to let runtimeStatsColl.Exists(p.ExplainID()) is true.
```

Comment:
May we remove all the mock info when tikv supports runtimeStats?

Reply:
It's removed now.

Reply:
1. Can we refactor this `if-else` to :
if runtimeStatsColl.ExistsRootStats(p.ExplainID()){
   row = append(row, runtimeStatsColl.GetRootStats(p.ExplainID()).String())
}else {
   row = append(row, runtimeStatsColl.CopSummary(p.ExplainID()))
}

2. Can we implement a `String()` function instead of `RuntimeStatsColl.CopSummary` for CopRuntimeStats? If so, we can use `row = append(row, runtimeStatsColl.GetCopStats(p.ExplainID()).String())`